module/data/item/consumable.mjs

```diff
@@ -102,6 +102,7 @@ export default class ConsumableData extends SystemDataModel.mixin(
     if ( this.type.value === "ammo" ) Object.entries(CONFIG.DND5E.itemProperties).forEach(([k, v]) => {
       if ( v.isPhysical ) valid.add(k);
     });
+    else if ( this.type.value === "scroll" ) valid.add("concentration");
```

Comment:
We should also add the other spell components other than material, because scrolls still might require verbal or somatic components.

```suggestion
    else if ( this.type.value === "scroll" ) CONFIG.DND5E.validProperties.spell
      .filter(p => p !== "material").forEach(p => valid.add(p));
```